SERVER_SIDE.md

```diff
@@ -71,6 +71,15 @@ var TomcatCookies = Cookies.withConverter({
 
 Alternatively, you can check the [Java Cookie](https://github.com/js-cookie/java-cookie) project, which integrates nicely with JavaScript Cookie.
 
+## Tomcat 8
```

Comment:
You can do something called **git amend**. Git has 2 states before commit, the modified state (when you modify a file), and the staged state (when you have staged your changes to commit). Assuming you are starting everything from scratch you will need to:

1. [Install Git for your OS](https://git-scm.com/) and make the `git` command available in the command line.
2. Choose a directory to setup the repository and run the command `git clone https://address-of-your-forked-repository.git`. It will create the `js-cookie` directory with the project structure there.
3. Edit the SERVER_SIDE file with the desired changes
4. Add the file to the staging area after you modified it using `git add SERVER_SIDE.md`
5. Run the command to amend the changes to the latest commit: `git commit --amend`

Then, after you have commited you just push the changes to your fork using the command `git push <remote> <source-branch>:<destination-branch>`. In this case it should be something like `git push origin patch-1:patch-1`

After you pushed, this Pull Request will update automatically.